temporal-testing/src/main/java/io/temporal/testing/TestActivityEnvironmentInternal.java

```diff
@@ -363,17 +363,13 @@ private Result executeActivity(
                       localActivity));
 
       try {
-        // 10 seconds is just a "reasonable" wait to not make an infinite waiting
-        return activityFuture.get(10, TimeUnit.SECONDS);
+        return activityFuture.get();
```

Comment:
Shouldn't the timeout be based on the activities timeout?

Reply:
I don't think anyone is using real activity timeouts in the TestActivityEnvironment.